### PR TITLE
Chapter 1

### PR DESCRIPTION
**Pull request by Ekzosta**
Я пропустив розділ про "Bitbucket", якщо воно потрібне - повернусь.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -50,5 +50,10 @@ group :development do
   gem 'spring-watcher-listen', '~> 2.0.0'
 end
 
+group :production do
+  gem 'pg', '0.20.0'
```
Comment (viktordanch) [inline Gemfile]: use postgresql for local database too. Do you have installed Postgres

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,3 +1,11 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def hello
+    render html: "hola, mundo!"
```
Comment (viktordanch) [inline app/controllers/application_controller.rb]: fix according to https://github.com/bbatsov/ruby-style-guide  (use ' if " no needed)

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+  end
+
+  def goodbye
+  	render html: "goodbye, world!"
```
Comment (viktordanch) [inline app/controllers/application_controller.rb]: fix according to https://github.com/bbatsov/ruby-style-guide (use ' if " no needed + tabulation)

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (Ekzosta) [reply app/controllers/application_controller.rb]: Fixed & pushed.

File: app/controllers/application_controller.rb (hunk 3 above)
Comment (Ekzosta) [reply app/controllers/application_controller.rb]: Fixed & pushed.

File: Gemfile (hunk 1 above)
Comment (Ekzosta) [reply Gemfile]: I haven't installed any DB. I use it only for the Heroku remote app (from Chapter 1.5). I didn't see the paragraph about installation local database, so i don't have PostgreSQL on local machine.
So, i need to install PostgreSQL?

Comment (viktordanch) [reply Gemfile]: we are using postgres on our projecs, so it's good task for you - postgres instalation
If you'll meet problems with it - write here

**Pull request merged**